### ISSUE TITLE
There is a MXNet implement of DenseNet with pretrained model in ImageNet dataset

### ISSUE THREAD

**Issue by miraclewkf** [liuzhuang13/DenseNet#35]
There is a MXNet implement of DenseNet with pretrained model in ImageNet dataset: https://github.com/miraclewkf/DenseNet

**Comment by liuzhuang13** [comment]
I've added this on our readme page. Thanks

**Comment by liuzhuang13** [comment]
Hi @miraclewkf , could you please make the models public? Some people have asked me about MXNet models, I think it would be really nice of you to give open access to the community. Thanks in advance.

**Comment by miraclewkf** [comment]
@liuzhuang13 Thanks for adding. I have put the pretrained model in Google Drive, everyone can download it from https://github.com/miraclewkf/DenseNet. You can emphasize that this implement has pretrained model in your readme page. Thanks.

**Comment by liuzhuang13** [comment]
@miraclewkf  Yes, I have put your link in the "pretrained model" section in the readme page.
But when I tried to download it, access needs to be requested. Are you aware of this?

**Comment by miraclewkf** [comment]
@liuzhuang13 Thanks. I update the link of Google Drive, try again please.
